gui/gui.js

```diff
@@ -1586,6 +1596,16 @@ class PearGUI extends ReadyResource {
 
   setSize ({ id, width, height }) { return this.get(id).setSize(width, height) }
 
+  setMinimizable ({ id, value }) {
+    this.message({ type: 'pear/setMinimize', data: value })
+    return this.get(id).setMinimizable(value)
```

Comment:
@HDegroote no the `get` method returns a "ghost" object with these funcs - what we should do at another point though is log out if the ghost ever gets used (because it shouldn't, it's backstop)